WeatherInfo/ViewControllers/Locations View Controller/LocationsViewController.swift

```diff
@@ -123,12 +123,15 @@ extension LocationsViewController: UITableViewDataSource {
         case .current:
             return 1
         case .favorite:
-            return max(favorites.count, 1)
+            return max(favorites.count, 0)
         }
     }
 
     func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
         guard let section = Section(rawValue: section) else { fatalError("Unexpected Section") }
```

Comment:
Instead of fatalError, now returning empty string for section header, in case section in not initialized